### PR TITLE
aligned mappings on BSD.

### PR DESCRIPTION
**Pull request by devnexen**
wrapper build fix (dlsym).

### CODE REVIEW

File: wrappers/mmapwrapper.h [hunk 1]
```diff
@@ -144,6 +144,10 @@ namespace HL {
       int fd = -1;
       startAddress = (char *) Alignment;
       mapFlag |= MAP_PRIVATE | MAP_ALIGN | MAP_ANON;
+#elif defined(MAP_ALIGNED)
+      int fd = -1;
+      size_t alignment = (sizeof(size_t) * sizeof(void *)) - 1ul - (size_t)__builtin_clzl(sz);
```
Comment (emeryberger) [inline wrappers/mmapwrapper.h:149]: This needs some explanatory comments...

**Review by emeryberger — CHANGES_REQUESTED** [review]
Hi - thanks for the PR. Please explain the issue it addresses and add comments to the code.

File: wrappers/mmapwrapper.h (hunk 1 above)
Comment (devnexen) [reply wrappers/mmapwrapper.h:149]: fair point :)

Comment (emeryberger) [reply wrappers/mmapwrapper.h:152]: Heap Layers already has an `ilog2` function (defined in `utility/ilog2.h`). I'm assuming this is the same functionality ? If so, please just use that function.

Comment (devnexen) [reply wrappers/mmapwrapper.h:152]: ah nice I did not see

**Pull request merged**